Lib/ast.py

```diff
@@ -1053,15 +1054,77 @@ def visit_AsyncWith(self, node):
         with self.block(extra=self.get_type_comment(node)):
             self.traverse(node.body)
 
+    def _str_literal_helper(
+        self, value, quote_types=("'", '"', '"""', "'''"), escape=""
+    ):
```

Comment:
(just a preference, feel free to ignore it)
```suggestion
    def _str_literal_helper(
        self, string, quote_types=("'", '"', '"""', "'''"), _custom_escapes=()
    ):
```